### DISCUSSION TITLE
How to get request IP?

### DISCUSSION THREAD

**Discussion by cawa-93** [nitrojs/nitro#533] [Q&A]
In my Nuxt app I have need to get visitor IP-address. I didn't found anythink about in docs. How I can get visitor IP?

**Comment by arashsheyda** [comment]
you can try something like this
```
 const req = event.node.req;
 const ip = req.headers["x-forwarded-for"] || req.socket.remoteAddress;
  ```
  
  or using npm packages such as [request-ip](https://www.npmjs.com/package/request-ip)
